dangerzone/cli.py

```diff
@@ -63,6 +65,8 @@ def cli_main(
 
     if getattr(sys, "dangerzone_dev", False) and dummy_conversion:
         dangerzone = DangerzoneCore(Dummy())
+    elif running_on_qubes() and os.environ.get("DZ_USE_CONTAINERS", "0") == "0":
```

Comment:
Two things here:

1. Let's document this option in the Qubes OS section of `INSTALL.md`, so that users are aware.
2. I propose moving the logic that checks for the envvar in the `running_on_qubes()` function, so that it's used by the GUI as well.

Reply:
1. :+1: 
2. I don't follow completely. Can you give an example?

Reply:
Sure. What I'm saying is that with the code above, only the Dangerzone CLI is affected. If you are in Qubes and you want to run the Dangerzone GUI with containers, you need the same code in the `gui/__init__.py` module.

Better yet, to avoid duplication, we can move this check (`os.environ.get("DZ_USE_CONTAINERS", "0")`) within `running_on_qubes`:

```python
  def running_on_qubes() -> bool:
      # https://www.qubes-os.org/faq/#what-is-the-canonical-way-to-detect-qubes-vm
      if os.environ.get("DZ_USE_CONTAINERS", "0") == "0":
          return os.path.exists("/usr/share/qubes/marker-vm")
      else:
          return False
```

Reply:
Done in 13aaeda. I also documented this on `BUILD.md`. We don't yet have a section for Qubes on `INSTALL.md`.